res/translations/es.json

```diff
@@ -0,0 +1,230 @@
+{
+    "A new version is available": "Una nueva versión está disponible",
+    "About": "Acerca de",
+    "Add Server": "Agregar servidor",
+    "Add to playlist": "Agregar a la lista de reproducción",
+    "Add to queue": "Agregar a la cola",
+    "Album": "Álbum",
+    "album": "álbum",
+    "Album count": "Número de álbumes",
+    "Album Count": "Número de álbumes",
+    "Album filters": "Filtros de álbumes",
+    "Album gain": "Ganancia del álbum",
+    "Album info not available": "Información del álbum no disponible",
+    "Album peak": "Pico del álbum",
+    "Albums": "Álbumes",
+    "albums": "álbumes",
+    "All": "Todo",
```

Comment:
What is the intended subject where this is used? The translation would be different depending on the genre of the subject as well as whether it is singular or plural. Some examples:
* All tracks: Todas las pistas.
* All albums: Todos los álbumes.
* All the collection: Toda la collección.

Reply:
This is where All is used, in the album filters UI, so I guess it should be plural and masculine (generos). I'll have to keep this in mind if I end up using the string "All" anywhere else in the interface:

<img width="255" alt="image" src="https://github.com/user-attachments/assets/cef3234b-5e9b-4381-9e18-f469d99a9ee3">